cocos/animation/marionette/animation-graph-context.ts

```diff
@@ -43,37 +44,59 @@ function findBoneByNameRecursively (from: Node, name: string): Node | null {
     return null;
 }
 
-/**
- * The binding context of an animation graph in layer-wide.
- */
-export interface AnimationGraphLayerWideBindingContext {
-    /**
-     * Indicates if the current layer is an additive layer.
-     */
-    additive: boolean;
-
-    /**
-     * The outer binding context.
-     */
-    outerContext: AnimationGraphBindingContext;
-}
-
```

Comment:
The `additive` flag is now integrated in `AnimationGraphBindingContext` so no longer need this context.